src/xbot/common/controls/sensors/XboxControllerWpiAdapter.java

```diff
@@ -0,0 +1,90 @@
+package xbot.common.controls.sensors;
+
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import edu.wpi.first.wpilibj.XboxController;
+import xbot.common.math.XYPair;
+
+public class XboxControllerWpiAdapter implements XXboxController {
+    
+    final XboxController internalGamepad;
+        
+    public enum XboxButtons {
+        A(1),
```

Comment:
Why are these manually forced to be 1-indexed even though their values are decremented by one for use?